### PR TITLE
Add apt-key

### PR DESCRIPTION
**Pull request by notpeter**

### CODE REVIEW

File: pages/linux/apt-key.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+`apt-key list`
+
+- Add a key to the trusted keystore
+
+`apt-key add {{public_key_file.asc}}`
+
+- Delete a key to the trusted keystore
+
+`apt-key del {{KEY_ID}}`
```
Comment (waldyrious) [inline pages/linux/apt-key.md]: Why uppercase?

File: pages/linux/apt-key.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# apt-key
+
+> APT key management utility for Debian and Ubuntu
```
Comment (waldyrious) [inline pages/linux/apt-key.md]: Maybe briefly describe what APT is?

**Comment by notpeter** [comment]
Fixed key_id snake case and include description of what APT is.

**Comment by igorshubovych** [comment]
I think the last example is so basic and common, that we should approve it, even though it is against Contributing Guidelines.

**Comment by notpeter** [comment]
I actually knew it was against the guidelines, but as its the most common use of apt-key, I figured it was worthy of an exception. 😜

**Comment by igorshubovych** [comment]
LGTM

File: pages/linux/apt-key.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- List trusted keys
+
+`apt-key list`
+
+- Add a key to the trusted keystore
+
+`apt-key add {{public_key_file.asc}}`
+
+- Delete a key to the trusted keystore
```
Comment (waldyrious) [inline pages/linux/apt-key.md]: Delete a key *from* ;)

File: pages/linux/apt-key.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# apt-key
+
+> Key management utility for APT Package Manager on Debian and Ubuntu
```
Comment (waldyrious) [inline pages/linux/apt-key.md]: I'd write "...for *the* APT Package Manager..."

**Comment by notpeter** [comment]
Fixed.

**Comment by waldyrious** [comment]
Thanks for the contribution and for bearing with us! :+1:

**Pull request merged**